server/routes/api.ts

```diff
@@ -1,15 +1,16 @@
 import * as Sentry from '@sentry/node';
 import { Router } from 'express';
-import { featureSwitches } from '../../shared/featureSwitches';
-import type { MembersDataApiResponse } from '../../shared/productResponse';
-import { isProduct, MDA_TEST_USER_HEADER } from '../../shared/productResponse';
+import { featureSwitches } from '@/shared/featureSwitches';
+import type { MembersDataApiResponse } from '@/shared/productResponse';
+import { isProduct, MDA_TEST_USER_HEADER } from '@/shared/productResponse';
```

Comment:
what does `@` do?

Reply:
It's just some syntactic sugar to make imports neater

Reply:
It's 2¢ on this, as I know we use it everywhere, but this does add "one-more-thing-you-need-to-know" and usually has it's own bespoke config which adds cognitive overhead over using standard config.

Reply:
Maybe? On the other hand it's quite a lot easier to read, the relative imports can look very messy and hard to parse.
I do agree the less magic approach is generally a good idea though

Reply:
yeah I'm not complaining, (as a non CS expert) I'm happy for us to use it, I was actually mainly worried it was an accidental change as it didn't seem releated to the main PR.

Reply:
It was giving me a warning so I changed it